pages/ksphere/kommander/1.2/Architecture/index.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
This is not related to this PR, but why is the Architecture capitalized in the URL path here? Everybody good with me changing this to lower case?

cc: @cmastr @russurquhart1 @pierrebeitz

Reply:
can't do this now @msdolbey. It'll break links. But let's open a JIRA please?